plugin/storage/es/options.go

```diff
@@ -210,6 +214,10 @@ func addFlags(flagSet *flag.FlagSet, nsConfig *namespaceConfig) {
 		nsConfig.namespace+suffixIndexPrefix,
 		nsConfig.IndexPrefix,
 		"Optional prefix of Jaeger indices. For example \"production\" creates \"production-jaeger-*\".")
+	flagSet.String(
+		nsConfig.namespace+suffixIndexDateSeparator,
+		defaultIndexDateSeparator,
+		"Optional date separator of Jaeger indices. For example \".\" creates \"jaeger-span-2020.11.20 \". Default: '"+defaultIndexDateSeparator+"'")
```

Comment:
you do not need to explain the default value in the description, it will be printed automatically

Reply:
Sorry @sniperking1234 , this was my suggestion. Thanks, @yurishkuro, I should've picked up on that!

Reply:
@albertteoh It doesn't matter. I've commit the code.